src/db/mongo.ts

```diff
@@ -0,0 +1,15 @@
+import mongoose from 'mongoose'
+import { mongoConfig } from '../config'
+
+export const createDBConnection = async () => {
+    try {
+        await mongoose.connect(`${mongoConfig.mongoUrl}`, {
```

Comment:
await is always used in calling funtion return the mongoose.connect here

Reply:
Yes. I have called this in the app.js file.